pkgs/by-name/va/vapoursynth-tedgemask/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  cmake,
+  pkg-config,
+  vapoursynth,
+  ninja,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vapoursynth-tedgemask";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "dubhater";
+    repo = "vapoursynth-tedgemask";
+    rev = "refs/tags/v${lib.versions.major finalAttrs.version}";
+    hash = "sha256-7ODhuW6UAG6TltuNNOWUWE9JbB6rXYcoGp/j7okXS5I=";
+  };
+
+  buildInputs = [ vapoursynth ];
+  nativeBuildInputs = [
+    meson
+    cmake
+    pkg-config
+    ninja # resolves: Could not detect Ninja
+  ];
+
```

```suggestion

    passthru.updateScript = nix-update-script { };

```

Requires adding `nix-update-script` to package inputs.